src/m365/spo/commands/listitem/listitem-set.ts

```diff
@@ -290,14 +292,12 @@ class SpoListItemSetCommand extends SpoCommand {
       }
       else {
         // Response is from /ValidateUpdateListItem POST call, perform get on updated item to get all field values
-        const returnedData: any = response.value;
-
-        if (!returnedData[0].ItemId) {
-          throw `Item didn't update successfully`;
-        }
-        else {
-          itemId = returnedData[0].ItemId;
+        const fieldValues: ListItemFieldValueResult[] = response.value;
+        if (fieldValues.some(f => f.HasException)) {
+          throw `The item was not updated successfully due to the following errors: ${os.EOL}${fieldValues.filter(f => f.HasException).map(f => { return `- ${f.FieldName} - ${f.ErrorMessage}`; }).join(os.EOL)}`;
```

Comment:
This error message is unclear. Let's rephrase to: "Updating the items has failed with the following errors"